docs/stylesheets/palette.css

```
@@ -0,0 +1,11 @@
+/*
+ * Active site palette. Change only the imported filename to switch themes:
+ *   palettes/burgundy.css   (default)
+ *   palettes/dusty-blue.css
+ *   palettes/terracotta.css
+ *   palettes/deep-navy.css   (design from 3eb8bd6)
+ */
+/* @import url("palettes/burgundy.css"); */
+/* @import url("palettes/dusty-blue.css"); */
+@import url("palettes/terracotta.css");
```

Stylelint (17.14.0)

[error] Expected "url("palettes/terracotta.css")" to be ""palettes/terracotta.css"" (import-notation)

(import-notation)